{{cookiecutter.project_name}}/.github/workflows/sync.yaml

```diff
@@ -0,0 +1,46 @@
+name: Sync Template
+
+on:
+    workflow_dispatch:
+    schedule:
+        - cron: "0 2 * * *" # every night at 2:00
+
+jobs:
+    sync:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Set up Python 3.10
+              uses: actions/setup-python@v2
+              with:
+                  python-version: "3.10"
+            - name: Install dependencies
+              # for now, pin cookiecutter version, due to https://github.com/cruft/cruft/issues/166
+              run: python -m pip install --upgrade cruft "cookiecutter<2" pre-commit
+            - name: Find Latest Tag
+              uses: oprypin/find-latest-tag@v1.1.0
+              id: get-latest-tag
+              with:
+                  repository: scverse/cookiecutter-scverse
+                  releases-only: false
+                  sort-tags: true
+                  regex: '^v\d+\.\d+\.\d+$' # vX.X.X
+            - name: Sync
+              run: |
+                  cruft update --checkout ${{ steps.get-latest-tag.outputs.tag }} --skip-apply-ask --project-dir .
+            - name: Create Pull Request
+              uses: peter-evans/create-pull-request@v4
+              with:
+                  commit-message: Automated template update from cookiecutter-scverse
+                  branch: template-update
+                  title: Automated template update from cookiecutter-scverse
+                  body: |
+                      A new version of the [scverse cookiecutter template](https://github.com/scverse/cookiecutter-scverse)
```

Comment:
Should we point to the releases of the template which hopefully also contains release notes?